internal/cri/config/config.go

```diff
@@ -325,6 +325,12 @@ type ImageConfig struct {
 
 	// StatsCollectPeriod is the period (in seconds) of snapshots stats collection.
 	StatsCollectPeriod int `toml:"stats_collect_period" json:"statsCollectPeriod"`
+
+	// Uses client.Pull to pull images locally, instead of containerd's Transfer Service
+	// By default it is set to false, i.e. use transfer service to pull images.
+	// When transfer service is used to pull images, pull related configs, like max_concurrent_downloads
+	// and unpack_config are configured under [plugins."io.containerd.transfer.v1.local"]
+	UseLocalImagePull bool `toml:"use_local_image_pull" json:"useLocalImagePull"`
```

Comment:
I feel we should at least print a warning (or error) if those fields are explicitly specified in cri and transfer service is enabled, otherwise they are sliencely ignored.